share/completions/krita.fish

```diff
@@ -0,0 +1,55 @@
+function __krita_append_image_format_value -a value current_token
+    string replace -r '^' -- $current_token $value |
+        string replace -r '$' ','
+end
+
+function __krita_complete_image_format
+    set -l previous_token (commandline -oc)[-1]
+    set -l current_token (commandline -t)
+
+    if test "$previous_token" = --new-image
+        switch $current_token
+            case '*,*,*'
+                # nothing is completed as arbitrary width and height are expected
+            case '*,'
+                __krita_append_image_format_value 'U8
+U16
+F16
+F32' $current_token
```

Comment:
So this would be `string join -- , $current_token U8 U16 F16 F32` - without the awkward line-breaks.